helm/cas-metabase/templates/deployment.yaml

```
@@ -16,6 +16,30 @@ spec:
     spec:
       imagePullSecrets:
         - name: dockerhub-registry
+{{- if index .Values "prod-test-restore" "enable" }}
+      initContainers:
+        - name: {{ .Chart.Name }}-wait-for-prod-restore
+          image: bash
+          imagePullPolicy: Always
+          command:
+            - bash
+            - -c
+            - |
+              set -euo pipefail;
+
+              while true; do
+                if [ -f /home/postgres/restore_completed ]; then
+                  exit 0
```

remove the file, then exit
```suggestion
                if [ -f /home/postgres/restore_completed ]; then
                  rm /home/postgres/restore_completed
                  exit 0
```